src/server/api/data.ts

```diff
@@ -0,0 +1,35 @@
+import axios from "axios";
+import "../config";
+
+interface Coordinates {
+  latitude: number;
+  longitude: number;
+}
+
+const API_KEY = process.env.TICKET_MASTER_API_KEY;
+
+export const getCoordinates = async (postcode: any): Promise<Coordinates> => {
+  const response = await axios.get(
+    `https://api.postcodes.io/postcodes/${postcode}`
+  );
+
+  const lat: number = response.data.result.latitude;
+  const long: number = response.data.result.longitude;
```

Comment:
You can use destructuring here to be a bit terser i.e. `const {latitude, longitude} = response.data.result`. You can even add the types , see [this article](https://flaviocopes.com/typescript-object-destructuring/)

Reply:
updated code - https://github.com/builtbybea/bass-and-treble/pull/2/commits/951304ea3c70afc6f336e0338dce0b7d9e847427